src/pg/test/expected/40_observatory_utility_test.out

```diff
@@ -0,0 +1,79 @@
+SELECT set_config(
```

Comment:
Oops, it looks like I didn't commit the test file. Will push once at a computer.